api/tests/Behat/features-v2/admin-management/edit-others.feature

```diff
@@ -0,0 +1,32 @@
+@v2 @admin-management
```

Comment:
I feel bad mentioning this as this is already a huuge amount of work and really detailed PR but should we be  validation checks? As in when entering wrong thing in fields, check we get validation messages?

Reply:
We should 😭  I shall add a validation step to the different parts that have it in.